src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/BulkCopyUtils.scala

```diff
@@ -160,6 +166,24 @@ object BulkCopyUtils extends Logging {
         conn.createStatement.executeQuery(queryStr)
     }
 
+    /**
+    * getComputedCols
+    * utility function to get computed columns.
+     * Use computed column names to exclude computed column when matching schema.
+    */
+    private[spark] def getComputedCols(
+        conn: Connection, 
+        table: String): List[String] = {
+        val queryStr = s"SELECT name FROM sys.computed_columns WHERE object_id = OBJECT_ID('${table}');"
+        val computedColRs = conn.createStatement.executeQuery(queryStr)
+        var computedCols : List[String] = List()
+        while (computedColRs.next()) {
+            val colName = computedColRs.getString("name")
+            computedCols = computedCols :+ colName
```

Comment:
Prefer "appended" here. ":+" is ok, but a esoteric which makes it hard to read.  Though its just a scala alias for "appended"

Reply:
I changed to ListBuffer to generate Computed Col list.